shared/templates/sudo_defaults_option/bash.template

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+if /usr/sbin/visudo -qcf /etc/sudoers; then
+    if ! grep -P "^(?!#)[\s]*Defaults.*[\s]+{{{ OPTION }}}[\s]*.*$" /etc/sudoers; then
+        cp /etc/sudoers /etc/sudoers.bak
+        echo "Defaults {{{ OPTION }}}" >> /etc/sudoers
+        if /usr/sbin/visudo -qcf /etc/sudoers; then
+            rm -f /etc/sudoers.bak
+        else
+            echo "Fail to validate remediated /etc/sudoers, reverting to original file."
+            mv /etc/sudoers.bak /etc/sudoers
```

Comment:
I think that you could add `false` here, so the last command ends with a non-zero return code and the whole remediation can then be evaluated as unsuccessful.

Reply:
That is a great suggestion, incorporated.